packages/dx-react-grid/src/plugins/table-column-reordering.jsx

```diff
@@ -25,7 +27,7 @@ const tableHeaderRowsComputed = (
   { tableHeaderRows },
 ) => tableHeaderRowsWithReordering(tableHeaderRows);
 
-export class TableColumnReordering extends React.PureComponent {
+class RawTableColumnReordering extends React.PureComponent {
```

Comment:
You create `GroupingPanelRaw` and `RawTableColumnReordering` names. Can we keep one style for naming `Raw` components. I think that postfix is better.

Reply:
Fixed.